java/curseforge/egg-curse-forge-generic.json

```
@@ -1,13 +1,13 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
     "meta": {
-        "version": "PLCN_v1",
+        "version": "PLCN_v2",
         "update_url": null
     },
-    "exported_at": "2025-05-27T11:05:16-07:00",
+    "exported_at": "2025-09-02T11:31:46+00:00",
     "name": "CurseForge Generic",
-    "author": "contact@chromozone.dev",
-    "uuid": "019bbf16-a3f3-470a-9c0b-f3995b5e032a",
+    "author": "alosh@alosh.shop",
+    "uuid": "1ef81a37-343d-43c6-b1b0-c4fb62965a2f",
```

```suggestion
    "author": "contact@chromozone.dev",
    "uuid": "019bbf16-a3f3-470a-9c0b-f3995b5e032a",
```
Author & uuid can never change.

---

i didnt change this, idk why it changed, it was exported from pelican